views/layout.erb

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
Your code is very clean and easy to read! I have nothing to add since I did not get that far. Looks perfect to me!